### PR TITLE
Login page branch

### LINKED ISSUE #36: Login Page
Creating a login page as the landing page of the project with hardcoded credentials:

username: admin
password: admin

username: developer
password: developer

### PR DESCRIPTION
**Pull request by nikunjdoshi67**
Added the components of the Login Page

"Closes #36"

### CODE REVIEW

**Comment by gagansk** [comment]
@nikunjdoshi67 

- There is no error message to indicate if the username or password is wrong
- How can a user logout?
- How will I be able to know whether I have logged in as an admin or a developer?

**Comment by nikunjdoshi67** [comment]
I have added the Logout Button 

The Admin and Developers can see the the type of user in the portal

Invalid Credentials Validation is handled now

**Comment by gagansk** [comment]
@skanthed A small merge conflict is present. Please fix it so that we can push it to the Master.

File: src/app/project_page/project_detail.tsx [hunk 1]
```diff
@@ -14,13 +16,15 @@ const ProjectDetail: React.FunctionComponent<{}> = () => {
     const [pod, setPod] = React.useState("")
     const [cpusage, setCpusage] = React.useState("")
 
-    const { projectId } = useParams();
+
+    let { projectId } = useParams();
     //console.log(projectId)
     React.useEffect(() => {
-        axios.get("https://7303ef19-853c-4092-a338-05502ff1eecd.mock.pstmn.io/projectlist/" + projectId).then(res => {
+        axios.get("https://6e905fc6-3dba-44d9-b37b-91bedb726dcf.mock.pstmn.io/projectlist/" + projectId).then(res => {
```
Comment (gagansk) [inline src/app/project_page/project_detail.tsx:23]: @skanthed Sync your PR by pulling in the changes from the remote master

**Review by gagansk — CHANGES_REQUESTED** [review]
Please pull the changes from remote master

File: src/app/project_page/project_detail.tsx (hunk 1 above)
Comment (skanthed) [reply src/app/project_page/project_detail.tsx:23]: The PR #56 will resolve this.

**Pull request merged**